TheTradeDeskCRMDestination.md

```diff
@@ -0,0 +1,110 @@
+
+If you select to hash the email addresses yourself, make sure to comply with the following requirements: 
+
+*  Remove leading and trailing spaces. 
+*  Convert all ASCII characters to lowercase. 
+*  In gmail.com email addresses, remove the following characters from the username part of the email address: 
+      *  The period (. (ASCII code 46)). For example, normalize jane.doe@gmail.com to janedoe@gmail.com. 
+      *  The plus sign (+ (ASCII code 43)) and all subsequent characters. For example, normalize janedoe+home@gmail.com to janedoe@gmail.com. 
+
+##Hash on Activation (#hash-on-activation)
+
+Data from unhashed namespaces is automatically hashed by Platform upon activation to a destination.  
+
+Attribute source data is not automatically hashed. When your source field contains unhashed attributes, check the Apply transformation option, to have Platform automatically hash the data on activation. The Apply transformation option is only displayed when you select attributes as source fields. It is not displayed when you choose namespaces. 
+
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (email or hashed email) used in the Trade Desk destination. |
+| Export frequency | **[!UICONTROL Daily Batch]** | As a profile is updated in Experience Platform based on segment evaluation, the profile (identities) are updated once a day downstream to the destination platform. Read more about [batch uploads](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#file-based).|
+
+{style="table-layout:auto"}
+
+
+## Connect to the destination {#connect}
+
+Before you can send, or activate, audience data to a destination, you must set up a connection to your own destination platform. While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination mapping in the future. 
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Advertiser ID]**: your Trade Desk Advertiser ID, which can either be shared by your TTD Account Manager or be found under Advertiser Preferences in the TTD UI. 
+*  **[!UICONTROL Account Type]**: Please choose the Existing Account option. 
+
+
+
+Setting a Data Governance policy is completely optional – please review Adobe’s [Data Usage Policy Overview](https://experienceleague.adobe.com/docs/experience-platform/data-governance/policies/overview.html?lang=en) for more details.  
+
+
```

Comment:
same comment as above: we only need one blank line between paragraphs. let's delete the extra blank lines, please.

Reply:
updated